content/hi/docs/reference/glossary/cidr.md

```diff
@@ -0,0 +1,17 @@
+---
+title: सीआईडीआर(CIDR)
+id: cidr
+date: 2019-11-12
```

Comment:
@rolfedh should we update the date as well with the current date?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
@sftim For localizations, do we preserve or reset the date?

Reply:
No opinion. We don't use that value anywhere when rendering the site. Individual localizations can pick what approach they want.